### PR TITLE
use uuid in the ID of EmptySticker

### PR DESCRIPTION
**Pull request by hjw4595**
The problem was that they could not make several empty stickers. The cause is key duplication. 
Therefore, for each sticker added, we used uuid to generate a unique ID.

### CODE REVIEW

File: src/templates/stickyboard-simple/src/components/base/PageBase.js [hunk 1]
```diff
@@ -107,38 +112,39 @@ class PageBase extends React.Component {
...
-            blocks: [{ i: 'EmptySticker' }, ...blocks],
-        });
+            blocks: [{i : EmptyStickerId } , ...blocks],
+        })
     };
 
     handleDelete = (id) => {
         const { blocks } = this.state;
-
+        console.log(id)
```
Comment (soaple) [inline src/templates/stickyboard-simple/src/components/base/PageBase.js:147]: Remove this code.

File: src/templates/stickyboard-simple/src/components/base/PageBase.js [hunk 2]
```diff
@@ -71,6 +75,7 @@ class PageBase extends React.Component {
             currentBreakpoint: 'lg',
             layout: undefined,
             blocks: undefined,
+            EmptyStickerId: uuid(),
```
Comment (soaple) [inline src/templates/stickyboard-simple/src/components/base/PageBase.js:78]: The variable name must start with lower case letter.

File: src/templates/stickyboard-simple/src/components/base/PageBase.js [hunk 3]
```diff
@@ -27,6 +27,10 @@ import MessageSnackbar from 'components/ui/MessageSnackbar';
 
 import Const from 'constants/Const';
 
+import  {  uuid  }  from  'uuidv4' ;
```
Comment (soaple) [inline src/templates/stickyboard-simple/src/components/base/PageBase.js:30]: Use single space only.

File: src/templates/stickyboard-simple/package.json [hunk 4]
```diff
@@ -45,7 +45,8 @@
         "request": "^2.88.2",
         "styled-components": "^4.4.1",
         "tls": "0.0.1",
-        "underscore": "^1.10.2"
+        "underscore": "^1.10.2",
```
Comment (soaple) [inline src/templates/stickyboard-simple/package.json:48]: Where is the 'underscore' used?

File: src/templates/stickyboard-simple/src/components/base/PageBase.js (hunk 2 above)
Comment (soaple) [reply src/templates/stickyboard-simple/src/components/base/PageBase.js:78]: Also, I think this variable is not needed.
Please use uuid() when the empty sticker inserted.

File: src/templates/stickyboard-simple/src/components/base/PageBase.js [hunk 5]
```diff
@@ -107,38 +112,39 @@ class PageBase extends React.Component {
     };
 
     handleInsert = () => {
-        const { layout, blocks } = this.state;
-
-        this.setState({
+        const {layout, blocks, EmptyStickerId} = this.state;
+        this.setState(
+            {
+            EmptyStickerId: uuid(),
```
Comment (soaple) [inline src/templates/stickyboard-simple/src/components/base/PageBase.js:118]: Please remove EmptyStickerId variable, and generate a new UUID every time when an empty sticker inserted.

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the comments.

File: src/templates/stickyboard-simple/package.json (hunk 4 above)
Comment (hjw4595) [reply src/templates/stickyboard-simple/package.json:48]: underscore is used in ./src/components/base/DrawerMenu.js, but I don't know why updated it.

**Comment by hjw4595** [comment]
after remove underscore there was an error(It's being used in the DrawerMenu.js) 
So I reinstalled and was updated as a result( package.json ).

File: src/templates/stickyboard-simple/package.json [hunk 6]
```diff
@@ -45,7 +45,8 @@
         "request": "^2.88.2",
         "styled-components": "^4.4.1",
         "tls": "0.0.1",
-        "underscore": "^1.10.2"
+        "underscore": "^1.11.0",
```
Comment (soaple) [inline src/templates/stickyboard-simple/package.json:48]: Please remove the package 'underscore'.

**Review by soaple — APPROVED** [review]
LGTM

**Pull request merged**